README.md

```diff
@@ -163,9 +163,13 @@ editing bindings. So, for example:
 * All your standard horizontal motion, selection, insertion, and
   deletion commands work as usual.
 * To delete your current input, just use `C-a C-k`.
-* To go up a directory, use `M-DEL` to delete it in the minibuffer.
+* To go up a directory, use `M-DEL` or `ESC C-backspace` (bound to
```

Comment:
Why `ESC C-backspace` instead of `C-M-DEL`?

Reply:
Oh, because `C-M-DEL` doesn't do the same thing somehow. I cannot say I understand why, but fair enough :P

Reply:
I will point out, by the way, that `backward-kill-sexp` doesn't actually move you one level up necessarily, if there are spaces in the path name.

Reply:
On Linux, `C-M-backspace` is sometimes set to kill the X server, which can be inconvenient. Using `ESC C-backspace` is therefore safer.

Reply:
> Oh, because C-M-DEL doesn't do the same thing somehow. I cannot say I understand why, but fair enough :P

I thought it isn't bound to that key by default because `(key-binding (kbd "C-M-DEL"))` returns `nil` for me. I have rebound it manually to that sequence. Maybe we should do that by default? If `C-M-DEL` kill the X server the user can rebind it.

Reply:
> I will point out, by the way, that backward-kill-sexp doesn't actually move you one level up necessarily, if there are spaces in the path name.

Thanks, we could fix that by modifying the syntax table when reading file names (adding space char as symbol constituents), would there be any downsides to this?